wrappers/Python/roadrunner/roadrunner.py

```diff
@@ -3733,13 +3733,29 @@ def simulate(self, start=None, end=None, points=None, selections=None, steps=Non
         if steps is not None:
             o.steps = steps
 
+        o.output_file = output_file
+
         result = self._simulate(o)
 
         o.steps = originalSteps
 
-        if result.shape[0] > Config.getValue(Config.MAX_OUTPUT_ROWS):
-            warnings.warn("Simulation returned more points than max output rows specified. "
-                          "Try incresing the number of maximum output rows or minimum step size.")
+        if 'override_max_rows' in r.getIntegrator().getSettings():
```

Comment:
'override_max_rows' was gotten rid of in a later commit, since it might be confusing to have this, 'output_file', and Config::MAX_OUTPUT_ROWS